cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,19 @@
+import {Upload} from "../page/index";
+
+const upload: Upload = new Upload();
+
+describe("Upload and download a file", ()=>{
+  let txt: {targetFile: string};
+  before(()=>{
+    cy.fixture("upload").then((texto)=>{
+      txt = texto;
+    });
+  });
```

Comment:
not sure if this is required

Reply:
Well, its a way to resolve a promise and obtain the value so he can use it outside the async flow, i don't mind

Reply:
thanks for the advise, It's already solved